README.md

```diff
@@ -229,9 +228,21 @@ will be used.
 
 Load the comma-separated list of tensorflow plugin/op libraries before conversion.
 
-#### --target
+#### --large_model
 
-Some models require special handling to run on some runtimes. In particular, the model may use unsupported data types. Workarounds are activated with ```--target TARGET```. Currently supported values are listed on this [wiki](https://github.com/onnx/tensorflow-onnx/wiki/target). If your model will be run on Windows ML, you should specify the appropriate target value.
+(Can be used only for TF2.x models)
+
+Only valid with parameter `--saved_model`. When set, creates a zip file containing the ONNX protobuf model and large tensor values stored externally. This allows for converting models that exceed the 2 GB protobuf limit.
+
+#### --continue_on_error
+Continue to run conversion on error, ignore graph cycles so it can report all missing ops and errors.
+
+#### --verbose
+Verbose detailed output for diagnostic purposes.
+
+#### --output_frozen_graph
+
+Saves the frozen and optimize tensorflow graph to file.
```

Comment:
Change to:

Save the frozen and optimized tensorflow graph to a file for debug.

Reply:
Done.